README.md

```diff
@@ -82,6 +82,29 @@ import '../../../../../test-tools/setup';
 
 Keywords from the testing frameworks can then be imported to your test files from `utils.ts`.
 
+### Functional End-to-End Testing
+
+Functional tests are run with [Testcafe](https://devexpress.github.io/testcafe/), which runs tests for each browser environment configured. The following browsers are currently tested:
+
+- Chrome
+- Firefox
```

Comment:
@ssbelle Eventually, but I think this ticket is fulfilled. We can just work to wrap it up.

You can add a comment to the ticket in Jira if you have items that you think should be done in the future.

Reply:
For the record: 

> @Daniel Moniz @Chantal https://devexpress.github.io/testcafe/documentation/extending-testcafe/browser-provider-plugin/#specifying-a-browser-for-a-test-task its definitely possible, we would have to decide what versions we want to cover(by listing in the config) and instead of using browsers on our locals possibly direct our testcafe code to use a service that has all those browser versions